eventbridge-bedrock-s3-aoss/example-pattern.json

```diff
@@ -0,0 +1,100 @@
+{
+  "title": "# EventBridge to Amazon Bedrock to Amazon OpenSearch Serverless",
```

Comment:
Pls give a meaningful pattern name that describes the functionality of this pattern. No need to add `#` at the beginning.

Reply:
done